lib/windows/winlean.nim

```diff
@@ -36,6 +36,8 @@ type
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
+  ULONG_PTR* = uint
```

Comment:
I dont not agree, because this "signed integers" making more problems, then benefits. Remeber last bug, where we can't use ports larger than 32768 because port was declared as int32 not uint32. So my opinion is when system using uint we must use uint too.